session_test.go

```diff
@@ -2394,47 +2394,3 @@ func (s *testSessionSuite) TestRetryResetStmtCtx(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(se.AffectedRows(), Equals, uint64(1))
 }
-
-func (s *testSessionSuite) TestCommitWhenSchemaChanged(c *C) {
```

Comment:
why remove this?

Reply:
This test is skipped and it's too simple.

Reply:
We've meet bug related to this, to avoid repeat the same mistake again, it should not be removed.